dev/Interop/CameraCaptureUI/CameraCaptureUI/CameraCaptureUI.h

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#pragma once
+#include "Microsoft.Windows.Media.Capture.CameraCaptureUI.g.h"
+#include "CameraCaptureUIPhotoCaptureSettings.h"
+#include "CameraCaptureUIVideoCaptureSettings.h"
+#include "TelemetryHelper.h"
+
+
+namespace winrt::Microsoft::Windows::Media::Capture::implementation
+{
+    struct token_and_path
+    {
+        hstring token;
+        hstring path;
+    };
+
+    struct CameraCaptureUI : CameraCaptureUIT<CameraCaptureUI>
+    {
+        CameraCaptureUI(winrt::Microsoft::UI::WindowId const& window) :
```

Comment:
Shouldn't window name be windowId? or Id? for improving code readability.

Reply:
Agreed, updated!